app/models/broker_agency_profile.rb

```diff
@@ -52,7 +52,7 @@ class BrokerAgencyProfile
     allow_blank: true
 
   validates :market_kind,
-    inclusion: { in: MARKET_KINDS, message: "%{value} is not a valid practice area" },
+    inclusion: { in: -> (val) { MARKET_KINDS }, message: "%{value} is not a valid practice area" },
```

Comment:
This is behaviorally the same but is evaluated at runtime instead of load. I needed this for RSpec